bin/templates/project/app/build.gradle

```diff
@@ -356,3 +377,7 @@ for (def func : cdvPluginPostBuildExtras) {
 if (hasProperty('postBuildExtras')) {
     postBuildExtras()
 }
+
+if (cdvHelpers.getConfigPreference('GoogleServicesEnable', 'false').toBoolean()) {
+    apply plugin: 'com.google.gms.google-services'
```

Comment:
In my plugin when I do this, for some reason it needs to be the full path to the plugin, not just the string... I think that might have had something to do with being a subproject though

```gradle
apply plugin: com.google.gms.googleservices.GoogleServicesPlugin
```